features/push/deep-linking.jsonnet

```diff
@@ -0,0 +1,38 @@
+local tags = import '../tags.libsonnet';
+
+[
+  {
+    id: 'handling-push-deep-link-start',
+    label: 'Handling push deep linking',
+    tag: tags.initTag,
+    log: 'Handling push notification deep link with payload: {payload} - pushClickBehavior: {behavior}',
```

Comment:
`pushClickBehavior ` is android only, no?

Reply:
Yes, this is purely for Android. iOS doesn't have anything similar to `pushClickBehavior`.